release.config.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+    branches: [
+        {
+            name: "master",
+            level: "minor",
+            dependencies: ["@zowe/perf-timing", "@zowe/imperative"]
+        },
+        {
+            name: "zowe-v?-lts",
+            level: "patch",
+            dependencies: ["@zowe/perf-timing", "@zowe/imperative"]
+        }
+        // {
+        //     name: "next",
+        //     prerelease: true,
+        //     dependencies: { "@zowe/perf-timing": "latest", "@zowe/imperative: "next" }
+        // }
+    ],
+    plugins: [
+        "@octorelease/changelog",
+        ["@octorelease/lerna", {
+            aliasTags: {
+                "latest": ["zowe-v2-lts", "next"]
```

Comment:
Q: Do we plan to remove the tag `next` once we decide to uncomment the `branches[2].name == "next"` 😋

Reply:
That's right, maybe we should add a comment here to remind us?

Reply:
yeah, a comment might be nice! 😋 
Thank you!